tests/cupyx_tests/scipy_tests/sparse_tests/test_csr.py

```diff
@@ -1648,3 +1648,98 @@ def test_mul_dense_vector(self, xp, sp):
             m * x
         # ...then perform the actual computation
         return m * x
+
+
+@testing.parameterize(*testing.product({
+    'dtype': [numpy.float32, numpy.float64],
+    'is_complex': [False, True],
+    'shape': [(4, 25), (10, 10), (25, 4)],
+    'nz_rate': [0.1, 0.5],
+    'opt': ['maximum', 'minimum'],
+}))
+@testing.with_requires('scipy')
+@testing.gpu
+class TestCsrMatrixMaximumMinimum(unittest.TestCase):
+
```

Comment:
Let's add different shapes test here to find errors

Reply:
Just to confirm, you mean to add shape combinations that raise the error?

Reply:
Exactly!